src/rulelib.cpp

```diff
@@ -71,9 +69,9 @@ int
 rules_init(const char *infile, int *nrules,
     int *nsamples, rule_t **rules_ret, int add_default_rule)
 {
-	FILE *fi;
-	char *line = NULL;
-    char *rulestr;
+	std::ifstream fi(infile);
+  std::string line;
```

Comment:
Indentation looks a bit strange here

Reply:
Did that fix it?

Reply:
Hmmm now it looks like some of them are indented with 2 spaces and some with 8...

Reply:
Ok I see what the problem was.